src/ncar-bufr2nc-fortran/kinds.f90

```diff
@@ -3,11 +3,12 @@ module kinds
 !                .      .    .                                       .
 ! module:   kinds
 !   prgmmr: treadon          org: np23                date: 2004-08-15
+!   prgmer: pnichols                                  data: 2023-03-09
 !
 ! abstract:  Module to hold specification kinds for variable declaration.
-!            This module is based on (copied from) Paul vanDelst's 
+!            This module is based on (copied from) Paul vanDelst's
 !            type_kinds module found in the community radiative transfer
-!            model
+!            model. Modifed to take
```

Comment:
- Some of this sentence is missing
- Could most/all of kinds.f90 be deleted, if the iso types are being used instead?